tox_core/src/onion/onion_announce.rs

```diff
@@ -703,7 +703,7 @@ mod tests {
             sendback_data: 42
         };
         let onion_return = OnionReturn {
-            nonce: secretbox::gen_nonce(),
+            nonce: [0; xsalsa20poly1305::NONCE_SIZE],
```

Comment:
```suggestion
            nonce: [42; xsalsa20poly1305::NONCE_SIZE],
```

Other tests use 42 as a pseudorandom value

Reply:
What do you mean?

Reply:
I mean `payload: vec![42; ONION_RETURN_3_PAYLOAD_SIZE]` ~ pseudorandom payload of 42*ONION_RETURN_3_PAYLOAD_SIZE. Whereas `nonce: [0; xsalsa20poly1305::NONCE_SIZE]` ~ empty value

Reply:
It does not matter, actually. It's up to you

Reply:
Got it. Changed for consistency.